IBAnimatable/AnimatableScrollView.swift

```diff
@@ -0,0 +1,154 @@
+//
+//  Created by Tom Baranes on 28/06/16.
+//  Copyright © 2015 Jake Lin. All rights reserved.
+//
+
+
+import UIKit
+import IBAnimatable
```

Comment:
My bad. I did that in my project then copy in IBAnimatable, forgot to remove it.